aksel.nav.no/website/components/sanity-modules/props/parts/PropTabell.tsx

```diff
@@ -28,22 +28,42 @@ const PropTable = ({ komponent }: PropTableProps) => {
 
         <dl>
           {komponent?.overridable && (
-            <DtList
-              prop={{
-                description: "OverridableComponent-api",
-                required: false,
-                name: "as",
-                type: "React.ElementType",
-              }}
-              parent={komponent?.title ?? ""}
-            />
+            <div className="border border-t-0 border-gray-300">
+              <dt className="px-2 py-2">
+                <Heading
+                  size="xsmall"
+                  level="4"
+                  className="inline-block rounded-medium bg-surface-alt-3-subtle px-1 font-mono text-small font-semibold"
+                >
+                  as?
+                </Heading>
+              </dt>
+              <dd>
+                <DtList
+                  prop={{
+                    description: "OverridableComponent-api",
+                    required: false,
+                    name: "as",
+                    type: "React.ElementType",
+                  }}
+                  parent={komponent?.title ?? ""}
+                />
+              </dd>
+            </div>
           )}
           {komponent?.propref?.proplist?.map((prop) => (
-            <DtList
-              key={prop.name}
-              prop={prop}
-              parent={komponent?.title ?? ""}
-            />
+            <div className="border border-t-0 border-gray-300" key={prop.name}>
+              <dt className="px-2 py-2">
```

Comment:
Synes det ble litt mye spacing mellom prop-navn og "Type:". Kanskje bottom padding kan sløyfes her?
```suggestion
              <dt className="px-2 pt-2">
```